.github/workflows/ci.yml

```diff
@@ -82,14 +82,14 @@ jobs:
         if: runner.os == 'linux'
         run: |
           sudo apt-get update --yes
-          sudo apt-get install --yes libjavascriptcoregtk-4.0-dev
+          sudo apt-get install --yes libjavascriptcoregtk-4.1-dev
 
       # See https://github.com/actions/runner-images/issues/8659
-      - name: Workaround Clang issue (GNU/Linux)
-        if: runner.os == 'linux' && matrix.platform.cc == 'clang'
-        run: |
-          sudo apt-get purge -y g++-13 gcc-13 libstdc++-13-dev
-          sudo apt-get install -y --allow-downgrades libstdc++-12-dev libstdc++6=12.* libgcc-s1=12.*
+      # - name: Workaround Clang issue (GNU/Linux)
+      #   if: runner.os == 'linux' && matrix.platform.cc == 'clang'
```

Comment:
This doesn't seem to be failing the CI anymore.
Let me know if this has any importance beyond the CI.